lib/request.js

```diff
@@ -66,7 +66,7 @@ function buildRequest (R, trustProxy) {
 }
 
 function buildRegularRequest (R) {
-  const props = [...R.props]
+  const props = R.props.slice()
```

Comment:
wasnt there something that says that spread is faster than slice?

Reply:
That array contains decorators, so it is empty most of the times or it does not have too much stuff:

object:

```
slice x 66,902,400 ops/sec ±0.98% (90 runs sampled)
spread x 46,305,568 ops/sec ±1.00% (94 runs sampled)
Fastest is slice
```

empty array:

```
slice x 175,975,861 ops/sec ±1.07% (83 runs sampled)
spread x 104,123,703 ops/sec ±1.07% (96 runs sampled)
Fastest is slice
```

<details>
  <summary>Click to expand!</summary>

```js
'use strict'
const Benchmark = require('benchmark')

const suite = new Benchmark.Suite()

const array = []
// const array = new Array(10).fill({ key: 'value', value: () => { } })

suite.add('slice', function () {
  return array.slice()
})

suite.add('spread', function () {
  return [...array]
})

// add listeners
suite.on('cycle', function (event) {
  console.log(String(event.target))
})
  .on('complete', function () {
    console.log('Fastest is ' + this.filter('fastest').map('name'))
  })
  // run async
  .run()

```
</details>